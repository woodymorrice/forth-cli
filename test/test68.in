."pi" 3.14 constant pi . . ."yum" set . .