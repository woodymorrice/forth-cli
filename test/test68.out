3.1400
stack empty
binary ops require at least 2 values on the stack
yum
stack empty
